sources/packages/web/src/components/aest/InstitutionUserSummary.vue

```diff
@@ -1,131 +1,126 @@
+      <EditInstitutionUser
+        :userType="userType"
+        :showEditUser="showEditUser"
+        :institutionUserName="institutionUserName"
+        @updateShowEditInstitutionModal="updateShowEditInstitutionModal"
+        @getAllInstitutionUsers="getAllInstitutionUsers"
+      />
     </div>
-    <ContentGroup class="mt-2">
-      <DataTable
-        :value="usersListAndCount.users"
-        :lazy="true"
-        :paginator="true"
-        :rows="DEFAULT_PAGE_LIMIT"
-        :rowsPerPageOptions="PAGINATION_LIST"
-        :totalRecords="usersListAndCount.totalUsers"
-        @page="paginationAndSortEvent($event)"
-        @sort="paginationAndSortEvent($event)"
-        :loading="loading"
+  </div>
+  <ContentGroup class="mt-2">
+    <DataTable
+      :value="usersListAndCount.users"
+      :lazy="true"
+      :paginator="true"
+      :rows="DEFAULT_PAGE_LIMIT"
+      :rowsPerPageOptions="PAGINATION_LIST"
+      :totalRecords="usersListAndCount.totalUsers"
+      @page="paginationAndSortEvent($event)"
+      @sort="paginationAndSortEvent($event)"
+      :loading="loading"
+    >
+      <template #empty>
+        <p class="text-center font-weight-bold">No records found.</p>
+      </template>
+      <Column
+        :field="UserFields.DisplayName"
+        header="Name"
+        sortable="true"
+      ></Column>
+      <Column :field="UserFields.Email" header="Email" sortable="true"></Column>
+      <Column :field="UserFields.UserType" header="User Type"></Column>
+      <Column :field="UserFields.Role" header="Role"></Column>
+      <Column :field="UserFields.Location" header="Locations"
+        ><template #body="slotProps">
+          <ul v-for="location in slotProps.data.location" :key="location">
+            <li>{{ location }}</li>
+          </ul></template
+        ></Column
       >
-        <template #empty>
-          <p class="text-center font-weight-bold">No records found.</p>
-        </template>
-        <Column
-          :field="UserFields.DisplayName"
-          header="Name"
-          sortable="true"
-        ></Column>
-        <Column
-          :field="UserFields.Email"
-          header="Email"
-          sortable="true"
-        ></Column>
-        <Column :field="UserFields.UserType" header="User Type"></Column>
-        <Column :field="UserFields.Role" header="Role"></Column>
-        <Column :field="UserFields.Location" header="Locations"
-          ><template #body="slotProps">
-            <ul v-for="location in slotProps.data.location" :key="location">
-              <li>{{ location }}</li>
-            </ul></template
-          ></Column
-        >
-        <Column :field="UserFields.IsActive" header="Status"
-          ><template #body="slotProps">
-            <StatusBadge
-              v-if="slotProps.data.isActive"
-              :status="GeneralStatusForBadge.Active"
+      <Column :field="UserFields.IsActive" header="Status"
+        ><template #body="slotProps">
+          <StatusBadge
```

Comment:
this can be written with single expression without v-if 

<StatusBadge
            :status="slotProps.data.isActive ? GeneralStatusForBadge.Active : GeneralStatusForBadge.InActive "
          />